alphalens/utils.py

```diff
@@ -205,29 +211,40 @@ def compute_forward_returns(factor_idx,
     Returns
     -------
     forward_returns : pd.DataFrame - MultiIndex
-        Forward returns in indexed by date and asset.
-        Separate column for each forward return window.
+        A MultiIndex DataFrame indexed by timestamp (level 0) and asset
+        (level 1), containing the forward returns for assets.
+        Forward returns column names must follow the format accepted by
+        pd.Timedelta (e.g. '1D', '30m', '3h15m', '1D1h', etc).
+        'date' index freq property must be set to a trading calendar (pandas
+        DateOffset), see infer_trading_calendar for more details.
+        This information is currently used only in cumulative returns
+        computation
     """
```

Comment:
I would change the last 4 lines like this: 
'date' index freq property (forward_returns.index.levels[0].freq) will be set to a trading calendar (pandas DateOffset) inferred from the input data (see infer_trading_calendar for more details).